### PR TITLE
[Console] Add completion to messenger:setup-transports command

### PR DESCRIPTION
**Pull request by Tayfun74**
| Q             | A
| ------------- | ---
| Branch?       | 5.4  
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no 
| Tickets       |  #43594 
| License       | MIT
| Doc PR        |  no 

- Add completion for command messenger:setup-transports (transport)

### CODE REVIEW

File: src/Symfony/Component/Messenger/Command/SetupTransportsCommand.php [hunk 1]
```diff
@@ -81,4 +83,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
         return 0;
     }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if (!$input->mustSuggestArgumentValuesFor('transport')) {
+            return;
+        }
+
+        $suggestions->suggestValues($this->transportNames);
```
Comment (wouterj) [inline src/Symfony/Component/Messenger/Command/SetupTransportsCommand.php]:
```suggestion
        if (!$input->mustSuggestArgumentValuesFor('transport')) {
            $suggestions->suggestValues($this->transportNames);

            return;
        }
```

While normally, we favor "early return" in the Symfony code base, I propose to not do that in the `complete()` methods. This way, we minimize the diff (and potential merge conflicts) when we add more completion suggestions to this method in a future version.

File: src/Symfony/Component/Messenger/Tests/Command/SetupTransportsCommandTest.php [hunk 2]
```diff
@@ -87,4 +88,22 @@ public function testReceiverNameArgumentNotFound()
         $tester = new CommandTester($command);
         $tester->execute(['transport' => 'not_found']);
     }
+
+    /**
+     * @dataProvider provideCompletionSuggestions
+     */
+    public function testComplete(array $input, array $expectedSuggestions)
+    {
+        /** @var MockObject&ServiceLocator $serviceLocator */
```
Comment (wouterj) [inline src/Symfony/Component/Messenger/Tests/Command/SetupTransportsCommandTest.php]:
```suggestion
```

**Review by wouterj — COMMENTED** [review]
Hi! Thank you for creating this PR.

I've left 2 minor suggestions on the code - other than that this looks ready to merge imho!

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Tayfun74.

**Pull request merged**